advanced/Scripts/piholeDebug.sh

```diff
@@ -271,12 +262,11 @@ checkProcesses() {
 	echo ":::"
 	echo "::: Logging status of lighttpd and dnsmasq..."
 	PROCESSES=( lighttpd dnsmasq )
-	for i in "${PROCESSES[@]}"
-	do
+	for i in "${PROCESSES[@]}"; do
```

Comment:
Note: This is a systemD only function. Dan you fool, you need to support `service` calls as well!